### DISCUSSION TITLE
loading state working when ever form element is adding value

### DISCUSSION THREAD

**Discussion by odion-cloud** [livewire/livewire#4550] [2. Show and tell]
in my app when ever i try to fill a form loading state is triggered, and when i select option select element , the loading is triggered, how do i stop this ? here is my global loading state in app.blade.js

                        <div x-data="{ loading: false }" x-show="loading" @loading.window="loading = $event.detail.loading">
                            <div class="page-contentloader"></div>
                        </div>

        <script>
          document.addEventListener('DOMContentLoaded', () => { 
              this.livewire.hook('message.sent', () => {
                  window.dispatchEvent(
                      new CustomEvent('loading', { detail: { loading: true }})
                  );
              } )
              this.livewire.hook('message.processed', (message, component) => {
                  window.dispatchEvent(
                      new CustomEvent('loading', { detail: { loading: false }})
                  );
              })
          });
      </script>

**Comment by joshhanley** [comment]
@odion-cloud loading will be showing as a message is sent when an input value changes. If you don't want that to happen, I'd suggest adding `.defer` to your select input which will not send a request to the server.

Hope this helps!
